icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2254,23 +2255,42 @@ class U_I18N_API NumberFormatterSettings {
     Derived usage(StringPiece usage) &&;
 
 #ifndef U_HIDE_DRAFT_API
-#ifndef U_HIDE_INTERNAL_API
+    /**
+     * Specifies the DisplayOptions. Such as `GrammaticalCase` specifies the desired case for
```

Comment:
```suggestion
     * Specifies the DisplayOptions. For example, UDisplayOptionsGrammaticalCase specifies the desired case for
```
By using the real name of the enum, and not putting it into back ticks, it will get a link in the API docs.

Reply:
done, thanks for the info.